cobalt/web/message_port.h

```diff
@@ -91,15 +93,31 @@ class MessagePort : public script::Wrappable,
                                              event_listener);
   }
 
-  web::EventTarget* event_target() { return event_target_; }
+  EventTarget* event_target() const { return event_target_; }
+  Context* context() const {
+    return event_target_->environment_settings()->context();
```

Comment:
do we need to check event_target_ exists before calling environment_settings()?

Reply:
It's only used in code where that check is already done, but that may not always be the case in the future, so adding that check can be useful and the processing overhead is trivial in this case.

Reply:
Thanks. Done.